ops/ops/interface_tls_certificates/model.py

```diff
@@ -14,5 +16,6 @@ class Data(BaseModel, extra=Extra.allow):
     """Databag from the relation."""
 
     ca: StrictStr = Field(alias="ca")
+    chain: Optional[StrictStr] = Field(None, alias="chain")
```

Comment:
Ahh cool, `None` is the automatic default if the field isn't provided as a kwarg

The `alias` isn't super important i don't think because it matches the name.  But for consistency sake it's okay

```suggestion
    chain: Optional[StrictStr] = Field(default=None, alias="chain")
```